deptofdefense/move.mil
# Add FAQ Content

I have…


## Summary of Changes

Addresses #72 .

Adds FAQ content, and organizes it by category instead of by tag.

Also fixes the tabs on the Moving Tips page so that navigating to that page with an anchor opens the corresponding tag.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. git checkout `faq-content`,
1. set up development dependencies according to [CONTRIBUTING.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTING.md),
1. run `bin/rails db:migrate`,
1. run `bin/rails db:seed`,
1. run `bin/rails server`, and
1. load up [http://localhost:3000](http://localhost:3000) in your Web browser of choice.

## Screenshots

### Before
<details><summary>Click to show screenshot</summary>

![fireshot capture 53 - frequently asked questions move mil - http___beta move mil_faqs](https://user-images.githubusercontent.com/26554826/30881390-b0023d38-a2d3-11e7-962e-7d0f705d21fd.png)

</details>

### After
<details><summary>Click to show screenshot</summary>

![fireshot capture 52 - frequently asked questions move mil - http___localhost_3000_faqs](https://user-images.githubusercontent.com/26554826/30881398-b502a994-a2d3-11e7-948f-90f5069dfae4.png)

</details>

### After (narrow screen)
<details><summary>Click to show screenshot</summary>

![fireshot capture 54 - frequently asked questions move mil - http___localhost_3000_faqs](https://user-images.githubusercontent.com/26554826/30881497-0de4215a-a2d4-11e7-83ba-391ecd9fc5c9.png)

</details>
